custom/enikshay/integrations/ninetyninedots/api_spec.py

```diff
@@ -59,24 +94,170 @@ def load_api_spec():
     return spec
 
 
+class MermParams(StrictJsonObject):
+    IMEI = jsonobject.StringProperty(required=False, exclude_if_none=True)
+    daily_reminder_status = jsonobject.StringProperty(required=False, exclude_if_none=True)
+    daily_reminder_time = jsonobject.StringProperty(required=False, exclude_if_none=True)  # HH:mm
+    refill_reminder_status = jsonobject.StringProperty(required=False, exclude_if_none=True)
+    refill_reminder_datetime = jsonobject.StringProperty(
+        required=False,
+        exclude_if_none=True
+    )  # yy/MM/dd HH:mm:ss
+    RT_hours = jsonobject.StringProperty(
+        required=False,
+        exclude_if_none=True
+    )  # 1 = 12 hours; i.e. for 3 days - RT_hours = 6
+
+
+class BasePatientPayload(StrictJsonObject):
+    sector = jsonobject.StringProperty(required=True)
+    state_code = jsonobject.StringProperty(required=False)
+    district_code = jsonobject.StringProperty(required=False)
+    tu_code = jsonobject.StringProperty(required=False)
+    phone_numbers = jsonobject.StringProperty(required=False)
+    treatment_supporter_name = jsonobject.StringProperty(required=False)
+    treatment_supporter_phone_number = jsonobject.StringProperty(required=False)
+    merm_params = jsonobject.ObjectProperty(MermParams, exclude_if_none=True)
```

Comment:
what's the difference between `ObjectProperty` and `SchemaProperty`

Reply:
I don't think jsonobject has a `SchemaProperty`: https://github.com/dimagi/jsonobject#jsonobjectobjectpropertyitem_type

Reply:
Oops. yeah that's a couchdbkit thing